### ISSUE TITLE
Breadcrumb Quests

### ISSUE THREAD

**Issue by ccnim** [ccshiro/cc-buglist#730]
We intend to redo how breadcrumb quests work. To accommodate for that we need to go through all of them. Looking for help compiling a list of all breadcrumb quests in vanilla and tbc.

Ex: http://www.wowhead.com/quest=9961/whats-wrong-at-cenarion-thicket
Basically a quest that leads you to a quest hub or quest NPC but does not require completion to pick up the quests at the new location.

**Comment by antepp** [comment]
What do you need help with? Code or testing`?

**Comment by Chulain** [comment]
## Eastern Kingdoms

if there are multiple wowhead links, the comments are usually found on the first link

---

## Dun Morogh

http://www.wowhead.com/quest=467/stonegears-search
2010/07/14 (Patch 3.3.5)
You won't be able to take this quest if you have already done the followup.

---

http://www.wowhead.com/quest=415/rejolds-new-brew
11/24/2007 (Patch 2.3.0)
This quest becomes undoable if you finish Shimmer Stout quest first. The second part of this quest chain can be done at the cost of skipping this quest (the first part)

---

http://www.wowhead.com/quest=419/the-lost-pilot
2010/10/24 (Patch 4.0.1)
For those who said they were unable to pick up this quest...

It is probably unavailable to anyone who has already done A Pilot's Revenge.

(thats the follow up)

---

## Duskwood

http://www.wowhead.com/quest=164/deliveries-to-sven
2008/09/04 (Patch 2.4.3)
If you are unable to pick up this quest and you are the right level for it, you most likely already went to Sven and picked up "Sven's Revenge". After abandoning the quest I was able to pick up Deliveries to Sven.
Not sure but I'm going to guess that if you've already completed and turned in Sven's Revenge, you wont be able to pick up this linkage quest.

---

http://www.wowhead.com/quest=165/the-hermit
 2008/06/13 (Patch 2.4.2)
This step of the Embalmer chain is optional. It's basically just a breadcrumb to lead you to Abercrombie. "Supplies from Darkshire" should be available even if you don't do it, and once you've completed that quest you won't be able to pick up this one.

---

http://www.wowhead.com/quest=163/raven-hill
2007/04/27 (Patch 2.0.12)
You can't do this quest if you've already done/accepted Jitters' Growling Gut

---

## Eastern Plaguelands

http://www.wowhead.com/quest=5601/sister-pamela
http://www.wowhead.com/quest=5142/little-pamela
2008/08/16 (Patch 2.4.3)
Little Pamela and Sister Pamela are mutually exclusive breadcrumb quests that lead you to Pamela. You can only complete one of them. However, you don't have to do either to access this quest. Once you've completed it, neither Little Pamela nor Sister Pamela will be available anymore.
(got comment here: http://www.wowhead.com/quest=5149/pamelas-doll)

---

http://www.wowhead.com/quest=5241/uncle-carlin
2007/10/11 (Patch 2.2.3)
Please note: You CAN already get "Defenders of Darrowshire" This is NOT a pre Quest for it.

---

http://www.wowhead.com/quest=5152/auntie-marlene
(most likely)

---

## Elwynn Forest

http://www.wowhead.com/quest=239/westbrook-garrison-needs-help
 04/14/2012 (Patch 3.1.3)
This quest is a breadcrumbs quest

---

http://www.wowhead.com/quest=783/deprecated-a-threat-within
(not sure)

---

http://www.wowhead.com/quest=3903/milly-osworth
(not sure)

---

## Eversong Woods

http://www.wowhead.com/quest=9254/the-wayward-apprentice
2011/10/29 (Patch 4.2.2)
But it is not a pre-requisite. If you have already encountered Apprentice Mirveda and picked up Corrupted Soil this quest will no longer be available.

---

http://www.wowhead.com/quest=9258/the-scorched-grove
not sure but i think it is

---

http://www.wowhead.com/quest=8888/the-magisters-apprentice
2009/12/10 (Patch 3.3.0)
This quest cannot be completed, if you have done the quests which are offered by Apprentice Loralthalis before.

---

http://www.wowhead.com/quest=9395/saltherils-haven
2008/01/30 (Patch 2.3.3)
To accept this quest, I had to abandon "The Party Never Ends", which you can accept before this quest.
2010/11/04 (Patch 4.0.1)
Breadcrumb quest

---

http://www.wowhead.com/quest=9253/runewarden-deryan
2009/12/10 (Patch 3.3.0)
If you have done Powering our Defenses before, you cannot accept this quest anymore.

---

http://www.wowhead.com/quest=9035/roadside-ambush
2009/09/14 (Patch 3.2.0)
Counts towards Loremaster of Eastern Kingdoms. This is significant because this quest is completely unavailable if you speak to Meledor before Ralen. Meledor will give you the next quest in the chain even if you didn't do this one, thus cheating you out of an extremely easy +1 to your Loremaster.

---

http://www.wowhead.com/quest=9358/ranger-sareyn
2009/11/24 (Patch 3.2.2)
You cannot get this quest if you have done Defending Fairbreeze Village before.

---

http://www.wowhead.com/quest=9359/farstrider-retreat
2010/01/15 (Patch 3.3.0)
Pre-req is completing Situation at Sunsail Anchorage.
Of course you could always skip this quest and go directly to Amani Encroachment.

---

http://www.wowhead.com/quest=9256/fairbreeze-village
2009/12/13 (Patch 3.3.0)
Also note that if you visited Fairbreeze Village already and got quests from Ranger Degolien you most likely cannot pick up this quest. (In similar vein to Darnassus and the person next to the portal who would normally point you to Priestess Amoora or whatever her name is.)

---

## Ghostlands

http://www.wowhead.com/quest=9877/a-restorative-draught
probably but not sure

---

http://www.wowhead.com/quest=9151/the-sanctum-of-the-sun
2010/05/01 (Patch 3.3.3)
This is an optional lead in quest, and becomes unavailable if you are on, or complete "War on Deatholme".

---

http://www.wowhead.com/quest=9282/the-farstrider-enclave
2010/09/10 (Patch 3.3.5)
indeed very frustrating since i am going for the Loremaster achievement, it would be nice to do this simple one but since i already got the follow up quest before i went to the sanctum of the sun for this quest. I cant get it anymore. :(

---

http://www.wowhead.com/quest=9144/missing-in-the-ghostlands
2008/10/04 (Patch 2.4.3)
You can not obtain this quest if you are either on, or have done the quest "The Fallen Courier" (http://www.wowhead.com/?quest=9147). If you are on this quest, abandoning it will open it up for you again.

---

http://www.wowhead.com/quest=9145/help-ranger-valanna
not sure

---

## Hillsbrad Foothills

http://www.wowhead.com/quest=494/deprecated-time-to-strike
(could be a breadcrumb for Battle of Hillsbrad)

---

## Ironforge

http://www.wowhead.com/quest=5090/a-call-to-arms-the-plaguelands
http://www.wowhead.com/quest=5091/a-call-to-arms-the-plaguelands
http://www.wowhead.com/quest=5066/a-call-to-arms-the-plaguelands
http://www.wowhead.com/quest=10373/a-call-to-arms-the-plaguelands
one for each city
see horde version, it's a breadcrumb

---

http://www.wowhead.com/quest=1453/reclaimers-business-in-desolace
(not sure)

---

http://www.wowhead.com/quest=2041/speak-with-shoni
2010/04/03 (Patch 3.3.3)
Breadcrumb quest. Not needed to acquire the follow-up quest, Underground Assault

(i wonder how many times i ran to IF to get this quest..)

---

http://www.wowhead.com/quest=8275/taking-back-silithus
2010/07/31 (Patch 3.3.5)  
This is one of the "talk to" quests, not necessary to do follow up. If you complete next quest before taking this one, you will lose +1 quest done for achievement.
(got comment from horde version)
wowhead shows in SW but used to be in IF

---

http://www.wowhead.com/quest=6721/the-hunters-path
(not sure about em tho)
also:
http://www.wowhead.com/quest=6072/the-hunters-path
http://www.wowhead.com/quest=6073/the-hunters-path
http://www.wowhead.com/quest=6722/the-hunters-path

---

http://www.wowhead.com/quest=1019/the-new-frontier
(not sure)
also:
http://www.wowhead.com/quest=1015/the-new-frontier
http://www.wowhead.com/quest=1047/the-new-frontier

---

http://www.wowhead.com/quest=3790/assisting-arch-druid-staghelm
http://www.wowhead.com/quest=10520/assisting-arch-druid-staghelm
http://www.wowhead.com/quest=3763/assisting-arch-druid-staghelm
http://www.wowhead.com/quest=3789/assisting-arch-druid-staghelm
one for each city
2008/11/03 (Patch 3.0.2)
if you pickup the quest from arch druid staghelm before you pickup this quest, then you won't get to pickup this quest.

---

http://www.wowhead.com/quest=2039/find-bingles
2007/07/11 (Patch 2.1.3)
well if you do Bingles quest first you will not be able to do this one.

---

http://www.wowhead.com/quest=707/ironband-wants-you
2010/12/01 (Patch 4.0.3)
I just picked up and turned in this quest, so it still exists. I got a 'You don't meet the requirements for that quest' after completion, presumably about the followup.
I guess some of the breadcrumb quests didn't quite get purged correctly?

probably a breadcrumb

---

http://www.wowhead.com/quest=3448/passing-the-burden
(not sure)

---

## Loch Modan

http://www.wowhead.com/quest=3448/passing-the-burden
06/10/2010 (Patch 3.1.3)
You wont be able to do this if you have done the next quest in the chain before this one..

---

http://www.wowhead.com/quest=1339/mountaineer-stormpikes-task
2009/02/28 (Patch 3.0.9)
If you've already accepted or completed "Stormpike's Order", you won't be able to get this quest.

---

## Redridge Mountains

http://www.wowhead.com/quest=244/encroaching-gnolls
(not sure)

---

## Silverpine Forest

http://www.wowhead.com/quest=428/lost-deathstalkers
2009/03/07 (Patch 3.0.9)
This quest becomes unavailable if you have completed the wild hearts quest. I abondonded the wild hearts quest and this became available.

---

## Stormwind City

http://www.wowhead.com/quest=1477/vital-supplies
05/31/2009 (Patch 3.1.2)
If you are after Loremaster, or even that bit of extra exp, make sure to pick up the breadcrumb quest, Vital Supplies, from Stormwind first! It will no longer be available after you complete this quest.

(found comment on next part)

---

http://www.wowhead.com/quest=6182/the-first-and-the-last
(not sure)

---

http://www.wowhead.com/quest=3765/the-corruption-abroad
12/30/2008 (Patch 3.0.3)
Remember, that this is the one of the "pointing" quests.
It is NOT a prerequisite for Researching the Corruption and it will NOT be available, if you have/have done Researching the Corruption.
I was lucky to have Researching the Corruption not even started yet, so I cancelled it and picked this one.

---

http://www.wowhead.com/quest=690/malins-request
10/03/2010 (Patch 3.1.3)
If you've done the followup in Arathi then this breadcrumb is no longer available to you.

---

http://www.wowhead.com/quest=3787/jonespyres-request
most likely, together with:
http://www.wowhead.com/quest=3788/jonespyres-request

---

http://www.wowhead.com/quest=1301/deprecated-james-hyal
02/04/2010 (Patch 3.1.3)
You cannot pickup this quest if you have already completed James Hyal from the npc Vincent Hyal in the Wetlands.
also:
http://www.wowhead.com/quest=1282/deprecated-they-call-him-smiling-jim
2007/06/05 (Patch 2.1.0)
You can't do this quest after turning in the last step of James Hyal (they're both just an excuse to get you to talk to Captain Vimes).

---

http://www.wowhead.com/quest=1097/elmores-task
02/26/2010 (Patch 3.1.3)
If you already completed Stormpike's Delivery, this quest won't be offered anymore.

---

## Stranglethorn Vale

http://www.wowhead.com/quest=638/trollbane
(most likely)

---

## Swamp of Sorrows

http://www.wowhead.com/quest=1418/neeka-bloodscar
06/08/2007 (Patch 2.1.1)
This quest seems to be unavailable if you take "Report to Helgrum" from Neeka Bloodscar in Kargath first.

---

http://www.wowhead.com/quest=9609/deprecated-help-watcher-biggs
 2008/09/03 (Patch 2.4.3)
It seems that this quest isn't available if you talk to Biggs directly, picking up Encroaching Wildlife. So if you want the XP for this, don't talk to Biggs first.

---

## Undercity

http://www.wowhead.com/quest=495/the-crown-of-will
04/28/2012 (Patch 3.1.3)
There appears no longer to be any follow up to this breadcrumb quest, even though the breadcrumb quest itself is still in game.

---

http://www.wowhead.com/quest=5961/the-champion-of-the-banshee-queen
(not sure)

---

http://www.wowhead.com/quest=3784/assisting-arch-druid-runetotem
same quest but horde version, so it's a breadcrumb
also:
http://www.wowhead.com/quest=936/assisting-arch-druid-runetotem
http://www.wowhead.com/quest=3762/assisting-arch-druid-runetotem

---

http://www.wowhead.com/quest=1004/the-new-frontier
(not sure)
also:
http://www.wowhead.com/quest=1018/the-new-frontier
http://www.wowhead.com/quest=1000/the-new-frontier

---

## Western Plaguelands

http://www.wowhead.com/quest=5093/a-call-to-arms-the-plaguelands
http://www.wowhead.com/quest=5094/a-call-to-arms-the-plaguelands
http://www.wowhead.com/quest=5095/a-call-to-arms-the-plaguelands
one for each city

11/24/2010 (Patch 3.1.3)
This is the Orgrimmar version of the breadcrumb quest leading to Scarlet Diversions in Western Plaguelands.

Please note that you will be unable to pick this quest up if you have already completed the follow-up and it will remain as incomplete in the Wowhead Quest Tracker.

---

## Wetlands

http://www.wowhead.com/quest=472/fall-of-dun-modr
(not sure)

---

http://www.wowhead.com/quest=10752/onward-to-ashenvale
http://www.wowhead.com/quest=990/trek-to-ashenvale
http://www.wowhead.com/quest=9432/travel-to-astranaar

 2009/12/07 (Patch 3.2.2)
This quest is completable by Dranei. However, you can only complete either this quest, or the quest Travel to Astranaar;  as they are both designed to introduce new players to Astranaar. Travel to Astranaar is a Dranei only quest, so if you are a Dranei and completed (or have in your quest log) that quest, you will not be able to pick this one up.

2010/04/03 (Patch 3.3.3)
Onward to Ashenvale may be done with Trek to Ashenvale. I picked up Onward to Ashenvale today immediately after picking up Trek to Ashenvale. I don't know what happens if you try to pick up one of these after completing the other though...

2008/05/02 (Patch 2.4.1)
This is another "time to go to the next leveling area" quest. As usual, simply follow the road.

the first one used to be a quest from wetlands but got changed later on, gna post it here as I don't have a Kalimdor list yet

---

http://www.wowhead.com/quest=305/in-search-of-the-excavation-team
(not sure)

---

http://www.wowhead.com/quest=473/report-to-captain-stoutfist
07/19/2010 (Patch 3.1.3)
This quest can be accidentally skipped by picking up war banners first. Once War Banners is done you can no longer go back and do this quest.

---

http://www.wowhead.com/quest=468/report-to-mountaineer-rockgar
06/10/2009 (Patch 3.1.3)
Appears as if you can't do this quest if you previously did "The Algaz Gauntlet" which I somehow managed to do without thinking. Won't mess you up too badly but you will miss out on a little bit of xp and rep.

---

http://www.wowhead.com/quest=463/the-greenwarden
(not sure)

**Comment by KimmoKM** [comment]
The warrior class quest breadcrumb [Speak with Ruga] (http://wowwiki.wikia.com/wiki/Quest:Speak_with_Ruga) is only available from Orgrimmar warrior trainer Sorek. According to Wowwiki it should also be available from Thunder Bluff, Brill and Undercity warrior trainers. However, a Wowhead comment, admittedly from 4.0.1, mentions he got the quest from Razor Hill so maybe all starting area and capital city warrior trainers should have it.
